/*
func sum(_ a: Int, _ b: Int) -> Int {
    let a = 32
    let result = a + b
    return result
}
 */
/*
var c = 1
if c == 1 {
    c += 1
    print(c)
} else {
    c += 1
    print(c)
}

func sum(_ a: Int, _ b: Int) -> Int {
    var c = 1
    c += 1
    let result = a + b + c
    return result
}

print(sum(1, 2))
print(sum(1, 2))
print(sum(1, 2))
print(sum(1, 2))
 */
/* В консоле мы увидим
 2
 5
 5
 5
 5
 Program ended with exit code: 0
 */
/*
 let a = 5
 let b = 4
 let result = a + b

 if a == b {
     print("а == b")
 } else {
     print("а != b")
 }


 func sum(_ a: Int, _ b: Int) -> String {
     let result = a + b
     if result == 2 {
         return "Двойка"
     } else {
         return "Не двойка"
     }
 }

 let sumResult = sum(a, b)
 print(sumResult)

 */
